### PR TITLE
net/fib: shell: fix warnings/errors thrown by clang on OS X

### PR DESCRIPTION
**Pull request by thomaseichinger**
Get RIOT compiling again for OS X.

### CODE REVIEW

**Comment by cgundogan** [comment]
untested ACK

File: sys/net/network_layer/fib/fib.c [hunk 1]
```diff
@@ -1283,7 +1283,7 @@ int fib_sr_get_route(fib_table_t *table, uint8_t *dst, size_t dst_size, kernel_p
         }
 
         if( skip ) {
-            if((*fib_sr == &table->data.source_routes->headers[i])) {
```
Comment (BytesGalore) [inline sys/net/network_layer/fib/fib.c]: oh :blush: , sry for this.
Also untested ACK from my side.

**Comment by thomaseichinger** [comment]
@emmanuelsearch Could you give a tested ACK?

**Comment by cgundogan** [comment]
tested ACK. Travis also agrees - GO

**Pull request merged**